Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -493,23 +493,25 @@ void MemoryWidget::OnSetValue()
   else
   {
     bool good_value;
-    u64 value = m_data_edit->text().toULongLong(&good_value, 16);
+    const auto& text = m_data_edit->text();
```

Comment:
`text()` returns a QString by value, so taking it by const reference, while valid, is somewhat visually misleading.